core/src/main/java/com/microsoft/applicationinsights/extensibility/context/ContextTagKeys.java

```diff
@@ -863,7 +880,9 @@ protected void reset(String name, String qualifiedName) {
         InternalSdkVersion = "ai.internal.sdkVersion";
         InternalAgentVersion = "ai.internal.agentVersion";
         SyntheticSource = "ai.operation.syntheticSource";
-        InternalNodeName = "ai.internal.nodename";
+        InternalNodeName = "ai.internal.nodeName";
```

Comment:
was the camel casing of nodeName intentional or this is typo?

Reply:
The correct version is "nodeName". See the same class in the .NET SDK.